marimbabot_vision/scripts/generate_hw_data.py

```diff
@@ -116,6 +121,20 @@ def render(sample):
 
 
 if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Hand-written data generation.")
+    parser.add_argument("--HW_SYMBOLS_DIR", type=str, required=False, help="Folder for the generated data.", default=HW_SYMBOLS_DIR)
+    parser.add_argument("--NUM_WORKER", type=int, required=False, help="Amount of workers that are used to generate the data.", default=NUM_WORKER)
+    parser.add_argument("--NAME_PREFIX", type=str, required=False, default=NAME_PREFIX)
```

Comment:
Add a help string for this one too.

Reply:
is it actually correct to call it prefix and not postfix?